test/functional/web-worker/test-amp-worker.js

```diff
@@ -39,15 +40,16 @@ describe('invokeWebWorker', () => {
     const fakeWorkerClass = () => fakeWorker;
     fakeWin = {Worker: fakeWorkerClass};
 
-    toggleExperiment(fakeWin, 'web-worker', true);
+    isExperimentOnStub = sandbox.stub(experiments, 'isExperimentOn');
```

Comment:
Test was failing due to a recent change in the implementation of `toggleExperiment` and figured stubbing is less brittle. Is there no good way to stub exported methods?

Reverted and changed to avoid the source of the original error.